### PR TITLE
Updates go to version 1.19

### PR DESCRIPTION
**Pull request by natalieparellano**
Signed-off-by: Natalie Arellano <narellano@vmware.com>

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Set up go
         uses: actions/setup-go@v3
         with:
-          go-version: '1.18'
+          go-version: '1.19'
```
Comment (jabrown85) [inline .github/workflows/build.yml:21]: Do we have a reason why we don't want to migrate to `go-version-file: 'go.mod'`?

**Review by jabrown85 — APPROVED** [review]
Looks good, with a question

File: .github/workflows/build.yml (hunk 1 above)
Comment (natalieparellano) [reply .github/workflows/build.yml:21]: No! Let's do it!

Comment (natalieparellano) [reply .github/workflows/build.yml]: Updated in 7cf318e3

**Review by jabrown85 — APPROVED** [review]

**Pull request merged**